src/kernels/gfx1030_36.HIP.fdb.txt

```diff
@@ -233,8 +233,8 @@
 1-1760-1760-1x1-1-1760-1760-1-0x0-1x1-1x1-0-NCHW-BF16-B=miopenConvolutionBwdDataAlgoGEMM:GemmBwd1x1_stride1,0.40441,0,miopenConvolutionBwdDataAlgoGEMM,not used;miopenConvolutionBwdDataAlgoDirect:ConvDirectNaiveConvBwd,7.61288,0,miopenConvolutionBwdDataAlgoDirect,not used
 1-1760-1760-1x1-1-1760-1760-1-0x0-1x1-1x1-0-NCHW-BF16-F=miopenConvolutionFwdAlgoGEMM:GemmFwd1x1_0_1,1.48587,0,miopenConvolutionFwdAlgoGEMM,not used;miopenConvolutionFwdAlgoDirect:ConvDirectNaiveConvFwd,5.48152,0,miopenConvolutionFwdAlgoDirect,not used
 1-1760-1760-1x1-1-1760-1760-1-0x0-1x1-1x1-0-NCHW-BF16-W=miopenConvolutionBwdWeightsAlgoDirect:ConvOclBwdWrW53,5.54679,0,miopenConvolutionBwdWeightsAlgoDirect,not used;miopenConvolutionBwdWeightsAlgoGEMM:GemmWrw1x1_stride1,364.46,0,miopenConvolutionBwdWeightsAlgoGEMM,not used
-1-1760-1760-1x1-1-1760-1760-1-0x0-1x1-1x1-0-NCHW-FP16-B=miopenConvolutionBwdDataAlgoGEMM:GemmBwd1x1_stride1,0.927561,0,miopenConvolutionBwdDataAlgoGEMM,1x1760x1760x1x1x1x1760x1760x1xNCHWxFP16x0x0x1x1x1x1x1xB;miopenConvolutionBwdDataAlgoWinograd:ConvBinWinogradRxSf3x2,4.38909,0,miopenConvolutionBwdDataAlgoWinograd,1x1760x1760x1x1x1x1760x1760x1xNCHWxFP16x0x0x1x1x1x1x1xB;miopenConvolutionBwdDataAlgoDirect:ConvDirectNaiveConvBwd,7.96897,0,miopenConvolutionBwdDataAlgoDirect,1x1760x1760x1x1x1x1760x1760x1xNCHWxFP16x0x0x1x1x1x1x1xB
```

Comment:
This file should not be changed,

Reply:
These changes are temporary for testing purposes. [642d384](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1510/commits/642d3848991a8daf82c11fc1a603c6f816a23cd9) commit will be reverted once testing is complete.